src/Microsoft.AspNet.Mvc.Core/Formatters/HttpTokenParsingRules.cs

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Text;
+
+namespace Microsoft.AspNet.Mvc.Formatters
+{
+    internal static class HttpTokenParsingRules
```

Comment:
Yes and yes, I just renamed the class as the methods left are more specific, but can undo the renaming